draco/analysis/flagging.py

```diff
@@ -416,6 +416,42 @@ def process(self, data):
         return out
 
 
+class BadGainMask(task.SingleTask):
+    """Get a mask of regions with bad gain.
+
+    Assumes that bad gains are set to 1.
```

Comment:
Would it be worth adding a `bad_gain_value` attribute, with default value of 1.0, in case there are situations in the future where some other number has been used to identify bad gains?